dm/tests/dmctl_command/run.sh

```diff
@@ -284,6 +285,44 @@ function run_validator_cmd {
 		"validator not found for task" 2
 }
 
+function run_validator_cmd_error() {
+	# status: without taskname
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation status --stage stopped" \
+		"Error" 1
+
+	# show errors: resolved error is illegal
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-errors --error resolved test" \
+		"Error" 1
+	# show errors: no task name
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-errors --error all" \
+		"Error" 1
+
+	# operate error: conflict id and --all flag
```

Comment:
case when 
- none of id or --all is given
- too many arguments
- non-number id

Reply:
ptal 3b38f05